content/authorization.md

```diff
@@ -940,8 +968,7 @@ their current authorizations.  Generally, such links
 are presented in conjunction with menu accessible from
 a status bar.
 
-For information on how to discover the management
-endpoint for a user, see
+For information on how to discover the management endpoint for a user, see
```

Comment:
Can you grab screenshots for the changes from here on down, please?

Reply:
Screen shots include formatting differences and the new question for a _suspended_ token.  This formatting coincides with the [Application Registration Prerequisites](https://fhir.cerner.com/authorization/application-registration-prerequisites/#faq), which was easier to read (with bold questions).

### BEFORE

![FHIR Authorization FAQ (before)](https://user-images.githubusercontent.com/104850375/168101752-07e1d3af-b7ed-46c1-9e22-6603df8af88f.png)

### AFTER

![FHIR Authorization FAQ (after)](https://user-images.githubusercontent.com/104850375/168101768-0f635f3b-4392-4dec-8c43-a396239f7b0d.png)